differ.js

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env node
+'use strict';
+var path = require('path');
+var elasticsearch = require('elasticsearch');
+var aws = require('aws-sdk');
+var program = require('commander');
+var awsAutoAuth = require('aws-auto-auth');
+
+var esVar = process.env.ELASTICSEARCH_PORT;
+var S3 = new aws.S3();
+var credentials = awsAutoAuth(aws);
+var esHost;
+var esPort;
+var esObj = {};
+var s3List;
+var makeRequest;
+
+if(credentials.accessKeyId){
+  makeRequest = S3.makeRequest.bind(S3);
+}else{
+  makeRequest= S3.makeUnauthenticatedRequest.bind(S3);
+}
+
+if(esVar){
+  esVar = esVar.split('//')[1].split(':');
+  esHost = esVar[0];
+  esPort = +esVar[1];
+}
+
+program
+  .version('0.0.1')
+  .option('-b, --bucket <bucket>', 'An S3 bucket where data resides.')
+  .option('-d, --directory <directory>', 'A directory within S3 or on the filesystem relative to where the differ was run.', '.')
+  .option('--alias <alias>', 'Elasticsearch index alias. Defaults to address.', 'address')
+  .option('-h, --host <host>', 'ElasticSearch host. Defaults to localhost unless linked to a Docker container aliased to Elasticsearch', esHost || 'localhost')
+  .option('-p, --port <port>', 'ElasticSearch port. Defaults to 9200 unless linked to a Docker container aliased to Elasticsearch.', Number, esPort || 9200)
+  .parse(process.argv);
+
+if(program.bucket && program.directory === '.') program.directory = '';
+
+var client = new elasticsearch.Client({
+  host: program.host + ':' + program.port,
+  log: []
```

Comment:
In the chronos script that differ was built for, its output is consumed line by line and fed into the ./grasshopper-loader binary. So logging to stdout would make this impossible.

Luckily, the combining loader changes this completely, and the new differ script will just call the loader via a module and no stdout parsing will take place, so logging will be restored.